### ISSUE TITLE
bug: Ionic Accessibility Multiple Issues

### ISSUE THREAD

**Issue by ShubhamDuggal** [ionic-team/ionic-framework#22788]
Issues:-
1. Ionic Toast messages are not readable by VoiceOver/TalkBack.
2. Ionic DatePicker not readable by VoiceOver/TalkBack.
3. Ionic Models not readable by VoiceOver/TalkBack.
4. Ionic Chip selected values are not announced by VoiceOver/TalkBack.
5. Ionic Menu not announced as "expanded or collapsed" by VoiceOver/TalkBack.
6. Ionic Input clearInput are announced as "reset search banner" button.

These issues are easily reproducible in any of the blank apps.

Please let me know by when it can be closed because we are planning to release the app by this month end
